autoload/codefmt/buildifier.vim

```diff
@@ -54,7 +54,13 @@ function! codefmt#buildifier#GetFormatter() abort
       " Parse all the errors and stick them in the quickfix list.
       let l:errors = []
       for line in split(v:exception, "\n")
-        let l:tokens = matchlist(line, '\C\v^stdin:(\d+):(\d+):\s*(.*)')
+        if empty(l:fname)
+          let l:fname_pattern = 'stdin'
+        else
+          let l:fname_pattern = escape(l:fname, '\')
```

Comment:
Alternatively we could just make the pattern start out something like `'\C\v[^:]+:\(\d+)...`, and assume that filenames never contained a `:`, but I suspect that wouldn't work everywhere.

Reply:
I could imagine corner cases with the filename being escaped/expanded/canonicalized that with this approach would lead to the error not being matched at all. It might be worth figuring out how to genericize it properly eventually (not a blocker since this does fix the major issue).

Reply:
I'm pretty sure that buildifier returns exactly the string we pass to it via `-path` (after all, it doesn't actually get to read the file content: that's piped to stdin). I'll commit this as-is, and we can always improve it if we need to later.